src/components/Form/FieldComboInput.tsx

```diff
@@ -114,6 +116,8 @@ export const FieldComboInput = <T extends FieldValues>({
         !price && !balance && errors && !errors[fieldName] && "pb-5",
         className && className
       )}
+      onClick={handleClick}
+      onKeyUp={handleClick}
```

Comment:
This doesn't make sense for me. Could you explain why we need click handler on non-interactive element?

Reply:
It does not actually bring trouble. Overall having `onClick` on `div` is incorrect in most cases and biome emphasize it.